webserver/views/user.py

```diff
@@ -98,7 +102,35 @@ def import_data():
                            lastfm_username=lastfm_username)
 
 
+@user_bp.route("/export", methods=["GET", "POST"])
+@login_required
+def export_data():
+    """ Exporting the data to json """
+    if request.method == "POST":
+        cassandra = webserver.create_cassandra()
+        filename = current_user.musicbrainz_id + "_lb-" + datetime.today().strftime('%Y-%m-%d') + ".json"
+
+        # Fetch output and convert it into dict with keys as indexes
+        output = []
+        for index, obj in enumerate(cassandra.fetch_listens(current_user.musicbrainz_id)):
```

Comment:
Just a general question about this approach: does this work reliably for users with a large (>100k entries) listen history? I've retrieved all my listens (~130000, a modified version of the JSON data is ~34MB) doing 1 request per second and with that approach I saw intermittent streaks of HTTP 502 (or 503 or 504, the timeout ones in any case) status codes.